uptane-standard.md

```diff
@@ -707,7 +707,7 @@ There may be several different filenames that all refer to the same image binary
 
 #### Send latest time to Secondaries {#send_time_primary}
 
-Unless the Secondary ECU has its own way of verifying the time or does not have the capacity to verify a time message, the Primary is CONDITIONALLY REQUIRED to send the time to each ECU. The Secondary will verify the time message, then overwrite its current time with the received time.
+The Primary SHOULD send the time to each ECU. 
```

Comment:
I feel like we're missing something by entirely removing the option for a Secondary to have it's own source of secure time, but maybe that's unrealistic anyway. I'll follow the insight of others.